cli/tests/e2e-pro/test_git.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from semgrep.git import get_project_url
+
+
+@pytest.mark.quick
+@pytest.mark.no_semgrep_cli
+@pytest.mark.osemfail
+def test_git_url_clean(mocker):
```

Comment:
maybe you can move this file in cli/tests/unit/ and
split the code in get_project_url() in two separate function, get_project_url() and clean_project_url
so then the test does not have to use mocker.patch.

Reply:
Thanks for the feedback, I considered whether that was a good idea so I appreciate the guidance. I'll work on that this week.